test/node/src/expression.test.ts

```diff
@@ -38,7 +38,7 @@ for (const dialect of DIALECTS) {
               or([
                 not(eb('first_name', '=', 'Jennifer')),
                 eb(eb('id', '+', 1), '>', 10),
-                eb(ref('id'), 'in', val([10, 20, 30])),
+                eb(ref('id'), 'in', [10, 20, 30].map(val)),
```

Comment:
let's not forget to mention the breaking change here.

Reply:
It's highly unlikely anyone has used that. I'd also argue that was a bugfix rather than a breaking change.